include/webserv.hpp

```diff
@@ -2,12 +2,20 @@
 #define WEBSERV_H
 
 #include <string>
-#include "Environment.hpp"
 #include <sstream>
 #include <cstdlib>
 #include <string.h>
+#include "Environment.hpp"
 #include "HTTPRequest.hpp"
 
+// #include "Environment.hpp"
```

Comment:
Do we need this commented out includes for any reason?

Reply:
Deleted it and took off any additional things (including Doxygen)